velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -157,6 +157,8 @@ std::shared_ptr<WriterProperties> getArrowParquetWriterOptions(
     properties =
         properties->data_page_version(arrow::ParquetDataPageVersion::V1);
   }
+  properties = properties->created_by(options.createdBy.value_or(
+      CREATED_BY_VERSION + std::string(" version ") + VELOX_VERSION));
```

Comment:
We don't need to set a default here again since arrow::WriterProperties::Builder sets the default?